lib/rucio/daemons/transmogrifier/transmogrifier.py

```diff
@@ -529,6 +531,7 @@ def run_once(heartbeat_handler: "HeartbeatHandler", bulk: int, **_kwargs) -> boo
                     #  Get all the rule and subscription parameters
                     rule_dict = __get_rule_dict(rule_dict, subscription)
                     weight = rule_dict.get("weight", None)
+                    ignore_availability = rule_dict.get("ignore_availability", None)
```

Comment:
Since it’s used below in `add_rule()`, the default should be `False`, not `None`. I’m surprised that our type checking didn’t catch this.